irohad/torii/impl/command_service.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2017 Soramitsu Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "torii/command_service.hpp"
+#include "common/types.hpp"
+
+namespace torii {
+
+  CommandService::CommandService(
+      iroha::model::converters::PbTransactionFactory &pb_factory,
```

Comment:
May be better solution is use smart pointers. Looks like work for unique_ptr. Or use copy instead.